packages/dds/tree/src/tableSchema.ts

```diff
@@ -69,14 +81,18 @@ export namespace TableSchema {
 		 */
 		const columnFields = {
 			id: schemaFactory.identifier,
-		} as const satisfies Record<string, ImplicitFieldSchema>;
+			props: schemaFactory.required(propsSchema),
+		} as const;
```

Comment:
I think it would be good to document why `satisfies Record<string, ImplicitFieldSchema>` was omitted here, since normally including it would be preferred.

Reply:
Added. The issue seems to be related to: https://github.com/microsoft/TypeScript/issues/52394